.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+logs/*
+resources/*
```

Comment:
Let's add empty line at the end file. Exclude folder resources/* from list. We need to see loj4j2.xml